pkg/storage/etcd/v3_helper.go

```diff
@@ -34,6 +33,7 @@ import (
 
 var (
 	ErrNotFound = errors.New("etcd: key not found")
+	ErrTestFail = errors.New("etcd: test failed")
```

Comment:
Sorry. Let me rectify my previous answer.

The correct solution should be we should have storage specific errors. Storage would be hiding etcd2 or etcd3 specific errors.